abc.templates/infra/contents/terraform.tf

```diff
@@ -31,4 +31,6 @@ terraform {
   }
 }
 
-provider "google" {}
+provider "google" {
+  user_project_override = true
```

Comment:
Using this always makes me worried about downstream effects, but I get the ease of setup with it.